src/index.coffee

```diff
@@ -12,6 +12,10 @@ app = (supported) ->
     do next
 
 class app.Locale
+  @substitutions =
```

Comment:
A comment here is probably a good thing. If not, at least in the commit message, but this is funky enough for one here.